pkg/archive/wrap.go

```diff
@@ -22,7 +23,7 @@ import (
 //
 // FIXME: stream content instead of buffering
 // FIXME: specify permissions and other archive metadata
-func Generate(input ...string) (Archive, error) {
+func Generate(input ...string) (io.ReadCloser, error) {
```

Comment:
Thanks - I see that `builder/remote.go` was updated but it looks like this function still returns an `io.ReadCloser`.